docs/source/data/data_catalog.md

```diff
@@ -386,7 +386,7 @@ CSVDataSet(
 
 ## Loading multiple datasets that have similar configuration
 
-You may encounter situations where your datasets use the same file format, load and save arguments, and are stored in the same folder. YAML has a [built-in syntax](https://yaml.org/spec/1.2.1/#Syntax) for factorising parts of a YAML file, which means that you can decide what is generalisable across your datasets so that you do not have to spend time copying and pasting dataset configurations in `catalog.yml`.
+You may encounter situations where your datasets use the same file format, load and save arguments, and are stored in the same folder. [YAML has a built-in syntax](https://yaml.org/spec/1.2.1/#Syntax) for factorising parts of a YAML file, which means that you can decide what is generalisable across your datasets so that you do not have to spend time copying and pasting dataset configurations in `catalog.yml`.
```

Comment:
Different datasets might use the same file format, load and save arguments, and be stored in the same folder. YAML has a [built-in syntax](https://yaml.org/spec/1.2.1/#Syntax) for factorizing parts of a YAML file, which means that you can decide what is generalizable across your datasets so that you need not spend time copying and pasting dataset configurations in `catalog.yml`.